api/v1/qdrantcluster_types.go

```diff
@@ -15,8 +15,18 @@ const (
 )
 
 const (
-	// The Annotation key to trigger a restart.
+	// RestartedAtAnnotationKey is the annotation key to trigger a restart.
+	// The annotation should be placed on the QdrantCluster instance.
+	// The value should be a [RFC3339 formatted] date.
+	// If the value is updated it will retrigger the restart.
+	// For historical reasons the key doesn't start with `operator.qdrant.com/`
 	RestartedAtAnnotationKey = "restartedAt"
+	// RecreateNodeAnnotationKey is the annotation key to recreate a certain node.
+	// The annotation should be placed on the pod created by the operator (for the node that need to be recreated).
+	// It is allowed to add this annotation to multiple pods, the operator will handle them all.
+	// The value should be 'true'
```

Comment:
I would relax the validation and allow any value here,  so that requester can use it for embedding additional semantic, like:

* reason for recreation
* timestamp when the request was made

The operator should remain unaware of the semantic and recreate the given node when the annotation is set. However, the operator should copy the annotation value in `.status` part along with the progress information, so that observers could figure out what is the status of the operation.

Reply:
Will relax it, like the suggestion.